dev/app-model.js

```diff
@@ -0,0 +1,213 @@
+                                @input=${this.handleChange}
+                                required></paper-input>
+                        </div>
+                        <div class="divide">
+                            <paper-input always-float-label 
+                                label="Description"
+                                type="text"
+                                name="description"
+                                .value="${this.description}"
+                                @input=${this.handleChange}
+                                ></paper-input>
+                        </div>
+                        <div class="divide">
+                            <paper-input always-float-label 
+                                label="Total Legands in Binding Group*"
+                                type="number"
+                                name="total"
+                                .value="${this.total}"
+                                @input=${this.handleChange}
+                                required></paper-input>
+                        </div>
+                    </div>
+                    <div class="full-input">
+                        <paper-input always-float-label 
+                            label="Comments"
+                            type="text"
+                            name="comment"
+                            .value="${this.comment}"
+                            @input=${this.handleChange}
+                            ></paper-input>
+                    </div>
+                </div>
+                </paper-dialog-scrollable>
+                <div class="bottom">
+                    <paper-button class="accept" raised autofocus @click="${this.handleConfirm}">Create</paper-button>
+                    <paper-button dialog-dismiss @click="${this.handleCancel}">Cancel</paper-button>
+                
+                </div>
+            </paper-dialog>
+        `;
+    }
+
+    toggleDialog(e) {
+        // console.log( this.shadowRoot);
+        this.shadowRoot.querySelector('#dialog').toggle();
+       
+        // this.$.dialog.toggle();
+    }
+
+    handleCancel() {
+        [this.name, this.legand, this.description, this.total, this.comment] = ['', 0, '', 0, ''];
+        // this.shadowRoot.querySelector('#name').validate().reset();
+        // console.log(this.name, this.legand, this.description);
+    }
+
+
+    handleConfirm(e) {
+        // console.log(this.name, this.description, this.legand);
+        let validated = false;
+        validated = this.shadowRoot.querySelector('#name').validate();
+        if(validated){
+            let myObj = {
+                name: this.name, 
+                legandsPromoted: this.legand, 
+                description: this.description,
+                totalLegandsInBindingGroup: this.total,
+                comment: this.comment
+            }
+            console.log(myObj);
+        }
+        
+        
+    }
+
+    handleChange(event) {
```

Comment:
you can try sending the parameter as handleChange(path, value) where value = event .target.value;
And while assigning the value simple do:
```
this[path] = value;
```